source/renderer/app/components/widgets/InView.tsx

```diff
@@ -0,0 +1,27 @@
+import { useRef, useState, useCallback } from 'react';
+
+export const useInViewPort = () => {
```

Comment:
Please also rename/relocate file accordingly :)

Reply:
That will be fixed once the task is unblocked.

Reply:
Updated, thanks.